src/assets/scripts/client/aircraft/FlightManagementSystem/LegModel.js

```diff
@@ -361,17 +361,11 @@ export default class LegModel {
      *
      * @for LegModel
      * @method _buildWaypointForHoldingPatternAtPosition
-     * @param routeString {string}
      * @param holdWaypointProps {object}
      * @return {array<WaypointModel>}
      */
-    _buildWaypointForHoldingPatternAtPosition(routeString, holdWaypointProps) {
-        const holdWaypointPropsWithPositionModel = this._buildHoldAtPositionWaypointProps(holdWaypointProps);
-        const waypointModel = new WaypointModel(holdWaypointPropsWithPositionModel);
-
-        return [
-            waypointModel
-        ];
+    _buildWaypointForHoldingPatternAtPosition(holdWaypointProps) {
+        return [new WaypointModel(holdWaypointProps)];
```

Comment:
Fixed.

Wonder why we even are returning an array like this... added a todo to figure that out and remove if possible.